src/structures.rs

```diff
@@ -149,10 +158,26 @@ impl fmt::Display for SvModuleDeclaration {
             write!(f, "{}", param)?;
         }
 
+        for instance in self.instances.clone() {
+            write!(f, "{}", instance)?;
+        }
+
         writeln!(f, "")
     }
 }
 
+impl fmt::Display for SvInstance {
+    fn fmt(&self, f: &mut fmt::Formatter) -> std::fmt::Result {
```

Comment:
Would you mind renaming `module_name` -> `module_identifier`, and `instance_name` -> `hierarchical_instance` to make it easier to match against the LRM.

See IEEE1800-2017 A.4.1.1 Module instantiation, bottom of the page.